.semaphore/semaphore.yml

```diff
@@ -61,6 +61,7 @@ blocks:
     jobs:
     - name: FV Test matrix
       commands:
+      - "! make check-wireguard"
```

Comment:
Please can you comment this; looks a bit odd to assert that wireguard is not supported.